Updated UG and DG to reflect changes in Person commands (#91)
AY2223S2-CS2103T-T12-3/tp: docs/UserGuide.md

> 

Reviewer: This is for line 65, change `list` to `listPerson`.

### Person Commands

### Adding a user: `addPerson`

> Adds a user to the records

Reviewer: ```suggestion
Adds a person to the ExpressLibrary.
```

Format: `addPerson n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`

<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
A person can have any number of tags (including 0)
</div>

Examples:

> * `addPerson i/A0123456M n/Bob Tan p/91112222 e/btan@mail.com b/A_Book`

Reviewer: ```suggestion
* `addPerson n/Bob Tan p/91112222 e/btan@mail.com a/ 313, Jurong East Street 32 b/Harry Potter`
```

### Deleting a person : `deletePerson`

> Delete a student given a student ID.

Reviewer: ```suggestion
Deletes a person at the specified person list index.
```

> Format: `deletePerson STUDENT_ID`

Reviewer: ```suggestion
Format: `deletePerson INDEX`
```

> * Deletes the person at the specified STUDENT_ID.

Reviewer: ```suggestion
* Deletes the person at the specified person list index given by the librarian.
```

* The index refers to the index number shown in the displayed person list.
* The index must be a positive integer 1, 2, 3, …​

Examples:
* `listPerson` followed by `deletePerson 2` deletes the 2nd person in the user records.

### Listing all users : `listPerson`

Shows a list of all users in the user records.

Format: `listPerson`

### Editing a person : `editPerson`

> Edits an existing person in the address book.

Reviewer: ```suggestion
Edits an existing person in the ExpressLibrary.
```

* Existing values will be updated to the input values.

* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.

* You can remove all the person’s tags by typing `t/` without
 specifying any tags after it.

Examples:

*  `editPerson 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.

*  `editPerson 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.

### Locating persons by name: `findPerson` 

[//]: # (_Details coming soon ..._)

Finds persons whose names contain any of the given keywords.

[//]: # ()
Format: `findPerson KEYWORD [MORE_KEYWORDS]`

[//]: # ()
* The search is case-insensitive. e.g `hans` will match `Hans`

* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`

* Only the name is searched.

[//]: # (* Only full words will be matched e.g. `Han` will not match `Hans`)

* Persons matching at least one keyword will be returned (i.e. `OR` search).

  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`

Examples:

> * `findPerson John` returns `john` and `John Doe`

Reviewer: Maybe we might want to change this example if findCommand is going to be updated.
```suggestion
* `findPerson John` returns `john` , `John Doe` and `johnston`.
```

### Clearing all entries : `clear`

[//]: # (_Details coming soon ..._)

> Clears all entries from the address book.

Reviewer: ```suggestion
Clears all entries from the ExpressLibrary.
```

> 

Reviewer: Could you help to remove line 191? I missed that, thanks.

Author: done
